core/src/main/scala/kafka/server/KafkaHealthcheck.scala

```diff
@@ -35,14 +36,15 @@ import org.apache.zookeeper.Watcher.Event.KeeperState
  * Right now our definition of health is fairly naive. If we register in zk we are healthy, otherwise
  * we are dead.
```

Comment:
Sorry to be lurking but with these additional metrics, is this comment still valid? 
```
* Right now our definition of health is fairly naive. If we register in zk we are healthy, otherwise
* we are dead.
```